Settings.cs

```diff
@@ -59,15 +63,7 @@ public string FilesOrigin
 		public int RetryCount { get; set; }
 
 
-		public HttpClient NewHttpClient(TimeSpan timeout)
-        {
-            return new HttpClient(GetHttpHandler())
-			{
-				Timeout = timeout
-			};
-		}
-
-		private HttpClientHandler GetHttpHandler()
+		public HttpClientHandler BuildHttpHandler()
```

Comment:
Should this method be public? Will it be used if someone overrides a http client?